src/django/api/templates/mail/claim_facility_submitted_body.html

```diff
@@ -2,59 +2,32 @@
 <html lang="en">
     <head>
         <title>
-            We received your request to claim an Open Supply Hub Facility
+            We received your request to claim a profile on Open Supply Hub
         </title>
     </head>
     <body>
         <p>
             Hi,
         </p>
         <p>
-            You're receiving this email because you were listed as the contact for a facility 
-            claim request on Open Supply Hub. We'll review the request and be in touch soon. The 
-            expected processing time for the initial review of a claim is 10 - 14 days. 
-        </p>
-        <p>
-            Vendors can refer facilities to claim their profiles, however, the Facility Claim request 
-            will only be approved when it is submitted by a facility owner and/or senior management staff 
-            at the facility location in question. If this claim is a referral, please visit <a href="https://opensupplyhub.org/auth/register">our website</a>,  
-            and register as a user on Open Supply Hub. 
-        </p>
-        <p>
-            Here is the information that was submitted with the request:
+           You are receiving this email because you submitted a request to claim the following production location on Open Supply Hub:
         </p>
         <ul>
             <li>
-                Facility: {{ facility_name }}, {{ facility_address }}, {{ facility_country }}
-            </li>
-            <li>
-                Facility URL: {{ facility_url }}
-            </li>
-            <li>
-                Contact person: {{ contact_person }}
-            </li>
-            <li>
-                Email: {{ email }}
-            </li>
-            <li>
-                Phone number: {{ phone_number }}
-            </li>
-            <li>
-                Company name: {{ company_name }}
```

Comment:
Since this information is no longer necessary, please delete it from the Python code. Please check the `send_claim_facility_confirmation_email` function.